docs/cookbook.rst

```diff
@@ -221,8 +233,11 @@ Would generate this log::
                         ... hoping it's equal to True.
```

Comment:
^ up on line 224 it looks like you missed a `::`.

If you just set it to `:`, the indented lines will be rendered as nested block quotes anyway, which might look nicer.

Reply:
I'm not sure I understand.  The double colon is so that when reading it should appear like this
![image](https://github.com/ScreenPyHQ/screenpy/assets/4211287/fe75e15f-d340-4e84-8b87-44d2d0e53308)

Reply:
What the, the `::` is supposed to interpret the next indented block like Python. I thought it would just have a bunch of red error outlining!